### ISSUE TITLE
[Bug] Can't publish

### ISSUE THREAD

**Issue by JohnGalt1717** [microsoft/kernel-memory#498]
### Context / Scenario

If you try and publish your dotnet project you get an error about llamasharp, which is a known issue with llamasharp.

I'm not using llamasharp, but it's included anyhow even in the .Core library.

 dotnet publish -c Debug -r linux-x64

### What happened?

Should build fine for publish.

### Importance

I cannot use Kernel Memory

### Platform, Language, Versions

all dotnet publish

### Relevant log output

```shell
Found multiple publish output file
s with the same relative path: C:\Users\k\.nuget\packages\llamasharp.backend.cpu\0.11.2\runtimes\linux-x64\native\avx\libllama.so, C:\Users\k\.nuget\packages\llam 
asharp.backend.cpu\0.11.2\runtimes\linux-x64\native\avx2\libllama.so, C:\Users\k\.nuget\packages\llamasharp.backend.cpu\0.11.2\runtimes\linux-x64\native\avx512\libll 
ama.so, C:\Users\k\.nuget\packages\llamasharp.backend.cpu\0.11.2\runtimes\linux-x64\native\libllama.so, C:\Users\k\.nuget\packages\llamasharp.backend.cpu\0.11.2\r 
untimes\linux-x64\native\avx\libllava_shared.so, C:\Users\k\.nuget\packages\llamasharp.backend.cpu\0.11.2\runtimes\linux-x64\native\avx2\libllava_shared.so, C:\Users 
\k\.nuget\packages\llamasharp.backend.cpu\0.11.2\runtimes\linux-x64\native\avx512\libllava_shared.so, C:\Users\k\.nuget\packages\llamasharp.backend.cpu\0.11.2\run 
times\linux-x64\native\libllava_shared.so.
```

**Comment by JohnGalt1717** [comment]
This bug is made even worse by the fact that there is no base package plus opt in packages for just what you need. The second you use .Core which isn't .Core at all, you get LLamaSharp whether you wanted it or not and everything else too.

I just need a root package with KernalMemoryBuilder and qdrant and I do the rest myself interfacing against LLama.cpp server with my own implementation.

If someone was using AzureAI, why would they need any of the other noise?  .Core should be .Core, and if you want to have a .All, then great.

**Comment by dluc** [comment]
Fixed in 0.60.240517.1
